apps/ai-content-generator/src/locations/ConfigScreenV1.tsx

```diff
@@ -25,6 +25,12 @@ const ConfigScreen = () => {
     const currentState = await sdk.app.getCurrentState();
     setApiKey('');
     setEditing(false);
+
+    if (parameters.key === '') {
+      sdk.notifier.error('A non-empty api key is required');
```

Comment:
Small copy suggestion. You can just say:

> A valid OpenAI API key is required

Reply:
Hmm - I avoided that because it only checks for empty string - it doesn't actually fully check if an API key is required but I'll make the changes regardless